hooks/use-bot-stats.ts

```
@@ -0,0 +1,402 @@
+        ...current,
+        count: current.count + 1 
+      })
+    })
+
+    const errorTypes: ErrorType[] = Array.from(errorTypeCounts.entries())
+      .map(([type, data]) => ({
+        type,
+        count: data.count,
+        percentage: (data.count / errorBots.length) * 100,
+        category: data.category,
+        priority: data.priority as any
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)